### PR TITLE
SQL: quotes escaping in collection name

### PR DESCRIPTION
**Pull request by unrealwork**
- [x] Tests
- [x] Fix missing issue comment

### CODE REVIEW

**Comment by KutuzovGregory** [comment]
lgtm

File: src/test/java/com/axibase/tsd/api/method/sql/examples/select/SqlExampleQuoteEscapingTest.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+        List<List<String>> expectedRows = Collections.singletonList(
+                Arrays.asList("tv1", "tv1", "tv2", "tv2", "tv3", "tv3")
+        );
+        assertTableColumnsNames(expectedColumnNames, resultTable);
+        assertTableRows(expectedRows, resultTable);
+    }
+
+
+    @Test
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/examples/select/SqlExampleQuoteEscapingTest.java]: please add issue note at head of each tests

**Comment by KutuzovGregory** [comment]
Remove Issue word in comments

**Comment by KutuzovGregory** [comment]
please use branch name like "surname_[issueNumber | some-description]

**Comment by KutuzovGregory** [comment]
review done

**Pull request merged**